site/astro.config.ts

```diff
@@ -1,10 +1,13 @@
+import mdx from "@astrojs/mdx";
 import react from "@astrojs/react";
 import solid from "@astrojs/solid-js";
 import tailwindcss from "@tailwindcss/vite";
 import { defineConfig } from "astro/config";
 
 // https://astro.build/config
 export default defineConfig({
+  srcDir: "src",
```

Comment:
`src` is the default value. I included it as a reminder because in the future I plan to move the site folder to the root and rename `src` to `site`.